database/neo4j/neo4j.go

```diff
@@ -73,24 +73,14 @@ func (n *Neo4j) Open(url string) (database.Driver, error) {
 	uri.Scheme = "bolt"
```

Comment:
Does this need to be changed to `neo4j`? See this table of [URI schemes](https://neo4j.com/docs/go-manual/4.2/client-applications/#go-driver-configuration-examples)

Reply:
Updated https://github.com/golang-migrate/migrate/pull/528/commits/8f1f62b1a65749ed853bc2d0390e23c9758f0457